test/functions/math_functions.test.cpp

```diff
@@ -0,0 +1,61 @@
+#include <string>
+#include "BigInt.hpp"
+#include "constructors/constructors.hpp"
+#include "functions/math.hpp"
+#include "operators/relational.hpp"
+#include "operators/binary_arithmetic.hpp"
+#include "functions/conversion.hpp"
+#include "third_party/catch.hpp"
+
+TEST_CASE("abs", "[conversion][string]") {
+	BigInt a("-12");
+	BigInt b("32");
+	BigInt c("-50");
```

Comment:
This file should be `test/functions/math.test.cpp`. It already exists. You can add just the test cases for `pow()`.

Reply:
Alright.